src/main/java/org/folio/circulation/support/utils/DateTimeUtil.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.circulation.support.utils;
+
+
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class DateTimeUtil {
+
+  private DateTimeUtil() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  public static ZonedDateTime atEndOfTheDay(ZonedDateTime dateTime) {
+    return dateTime
+      .withHour(23)
+      .withMinute(59)
+      .withSecond(59);
+  }
+
+  public static boolean isLongTermPeriod(ChronoUnit chronoUnit) {
```

Comment:
Can this decision be part of the `TimePeriod` class which is used to define the expiration period?